### PR TITLE
Fix AMP export of Twitter tweets

### PR DESCRIPTION
**Pull request by nicolas-fricke**
The `amp-twitter` tag expects an `data-tweetid` attribute containing the ID of a tweet. For twitter embed elements in the `article_json` format we store `"<handle>/<tweet_id>"` as the `embed_id`. Therefore, we have to extract the tweet ID and only pass that part over to the `amp-twitter` tag.

### CODE REVIEW

**Review by dsager — APPROVED** [review]

**Comment by christofdamian** [comment]
+1

**Comment by nicolas-fricke** [comment]
Rebased to resolve merge conflicts ☝️

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 ## WIP
 - Start work on Facebook Instant Article exporter
+<<<<<<< HEAD
```
Comment (miguelbernadi) [inline CHANGELOG.md]: Oops?

File: spec/article_json/export/amp/elements/embed_spec.rb [hunk 2]
```diff
@@ -1,11 +1,17 @@
 describe ArticleJSON::Export::AMP::Elements::Embed do
   subject(:element) { described_class.new(source_element) }
   let(:source_element_embed_type) { :youtube_video }
+  let(:source_element_embed_id) { '666' }
   let(:source_element) do
     ArticleJSON::Elements::Embed.new(
       embed_type: source_element_embed_type,
+<<<<<<< HEAD
```
Comment (miguelbernadi) [inline spec/article_json/export/amp/elements/embed_spec.rb]: Oops?

File: spec/article_json/export/amp/elements/embed_spec.rb [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
       embed_id: 666,
       caption: caption,
+=======
```
Comment (miguelbernadi) [inline spec/article_json/export/amp/elements/embed_spec.rb]: Oops?

File: spec/article_json/export/amp/elements/embed_spec.rb [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
+      embed_id: source_element_embed_id,
+      caption: [ArticleJSON::Elements::Text.new(content: 'Foo Bar')],
+>>>>>>> Fix AMP export of Twitter tweets
```
Comment (miguelbernadi) [inline spec/article_json/export/amp/elements/embed_spec.rb]: Oops?

**Comment by nicolas-fricke** [comment]
@miguelbernadi Oops, indeed – that was a pretty bad merge I did there.. Should be fixed now :)

**Comment by towanda** [comment]
+1

**Pull request merged**